addons/mail/static/src/messaging/entity/thread_viewer/thread_viewer.js

```diff
@@ -191,14 +221,44 @@ function ThreadViewerFactory({ Entity }) {
             });
 
             if (thread && this.thread !== thread) {
+                if (this._loaderTimeout) {
+                    clearTimeout(this._loaderTimeout);
+                }
                 this.link({ thread });
                 if (!this.threadCache.isLoaded && !this.threadCache.isLoading) {
                     this.threadCache.loadMessages();
                 }
             }
             if (this.threadCache !== prevThreadCache) {
+                if (this._loaderTimeout) {
+                    clearTimeout(this._loaderTimeout);
+                }
                 this._writeComponentHint('change-of-thread-cache');
             }
+
+            if(!this.threadCache) {
+                this._write({
+                    hasLoader:false,
```

Comment:
Usually, boolean attributes prefixed with "has" affects the scope of logic, while "is" fine-tunes current scope of logic.
Here an "is" fits best. Suggest renaming to "isShowingLoading".

Reply:
I really don't spot the difference but alright I'll change ..